lib/devcycle-ruby-server-sdk/localbucketing/event_queue.rb

```diff
@@ -31,6 +31,11 @@ def initialize(sdkKey, options, local_bucketing)
       nil
     end
 
+    def close
```

Comment:
the `cleanup` method at the bottom is supposed to do the same thing as `close`. I named it after what the nodejs sdk had. Since they're both meant to do the same thing, you should remove one of them

Reply:
https://github.com/DevCycleHQ/go-server-sdk/blob/main/client.go#L455
I was going off Go as the base for this; will remove cleanup.

Reply:
sounds good to me!